pkg/handler/handler.go

```diff
@@ -68,7 +69,9 @@ func defaultHandle(ctx context.Context, event sdk.Event, run runner.Runner) erro
 
 	// iterate events from ansible, looking for the final one
 	statusEvent := eventapi.StatusJobEvent{}
+	eHandler := events.NewLoggingEventHandler(events.Tasks)
```

Comment:
It strikes me that "logger" could be a more descriptive name for this thing. It logs events. The fact that its method is named `Handle` is incidental; it could just as easily be called `Log` to emphasize what it does, `Info` to make clear what level it will use, `Process` because that's a verb that's no less descriptive than `Handle`, etc.

Reply:
This is a stop gap, eventually, the handler should take a list of `EventHandlers`. A user could define their own `EventHandler` or we could create more. Specifically, we could create an `EventRecorderHandler` which will publish the events.

I was thinking that keeping the interface and variables around this `EventHandler` interface generic would better. Thoughts?